### DISCUSSION TITLE
Core Team Maintainers to update and maintain project.

### DISCUSSION THREAD

**Discussion by ntegral** [ntegral/nestjs-sentry#116] [Announcements]
Thanks for supporting the ntegral/nestjs-sentry project.  Please let me know if you are interested in being a part of the core team to maintain this project.  Note: all code will need to past all tests before merging with the main.

**Comment by H4ad** [comment]
I like Sentry and Nest, I think I give some help to maintain this package.

I'm really interested in see the support for nest v10.
